python/Ganga/GPIDev/Credentials/ICredential.py

```diff
@@ -524,3 +524,4 @@ def timeleftInHMS(self, force_check=False):
         """
         logger.warning("Dummy method used - no information returned")
         return ""
+
```

Comment:
This has to be a problem between editors as it doesn't appear in my vim, I'll revert the change as it's not relevant to what's going on